docs/rules/no-url-protocols.md

```diff
@@ -2,9 +2,15 @@
 
 Rule `no-url-protocols` will enforce that protocols and domains are not used within urls.
 
+## Options
+
+* `protocol-relative-urls`: `true`/`false` (defaults to `false`)
+
```

Comment:
A small message here about the fact the `no-domains` rule could be used instead and that this option/functionality will be deprecated in sass-lint 2.0 would be good.